File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int>seen;
        for(int i=0;i<nums.size();i++){
            int rem=target-nums[i];
            if(seen.find(rem)!=seen.end()) return {seen[rem],i};
            seen[nums[i]]=i;
        }
        return {};
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        collection={}
        for i in range(len(nums)):
            req=target-nums[i]
            if req in collection:
                return [collection[req],i]
            collection[nums[i]]=i

        